### PR TITLE
Allow restorations with no plugins. 

### LINKED ISSUE #96: Allow saving without custom plugin
Right now we need to have a custom plugin before any of the server functions work. We shouldn't have to have a custom plugin on the client to save.

image

@garrettluu says:

> figured it out! this bug ONLY happens when we have no custom plugin
> so it probably does something weird when trying to encode an empty string

See #93.

### PR DESCRIPTION
**Pull request by penguingovernor**
This PR should fix #96 by using the `||` operator. Passing null to `{decode,encode}Base64` causes the error described in the issue. Using the `||` we can always pass an empty string in the case the plugin isn't there.

### CODE REVIEW

File: src/state/REPLState.js [hunk 1]
```diff
@@ -77,11 +77,11 @@ class REPLState {
   static Decode(encodedState) {
     let jsonState = JSON.parse(encodedState);
     return new REPLState(
-      decodeBase64(jsonState.base64SourceKey),
-      decodeBase64(jsonState.base64PluginKey),
+      decodeBase64(jsonState.base64SourceKey || ""),
+      decodeBase64(jsonState.base64PluginKey || ""),
```
Comment (wlawt) [inline src/state/REPLState.js:81]: image

Tried performing the forking procedure, is this supposed to happen (missing the fork icon)?

Comment (penguingovernor) [reply src/state/REPLState.js:81]: Just checked out master same issue 🙁

Comment (wlawt) [reply src/state/REPLState.js:81]: @penguingovernor should be fixed now after #103 has been merged into `master`

**Review by wlawt — APPROVED** [review]
Works LGTM!

**Pull request merged**